buildspec/release/50githubrelease.yml

```diff
@@ -34,10 +34,11 @@ phases:
             - echo "Writing hash to $HASH_UPLOAD_TARGET"
             - echo $HASH > $HASH_UPLOAD_TARGET
             - echo "posting $VERSION with sha384 hash $HASH to GitHub"
-            - RELEASE_MESSAGE="AWS Toolkit for VS Code $VERSION"
+            - PKG_DISPLAY_NAME=$(grep -m 1 displayName packages/${TARGET_EXTENSION}/package.json | grep -o '[a-zA-z][^\"]\+' | tail -n1)
+            - RELEASE_MESSAGE="${PKG_DISPLAY_NAME} for VS Code $VERSION"
             - |
-                if [ "$STAGE" = "prod" ] && [ "$TARGET_EXTENSION" = "toolkit" ]; then
```

Comment:
due to this change we should wait to merge this until the auto release pipeline is disabled.